Updating baskets for new challenges

ekosmin opened the pull request:
A handful of changes to baskets:
* Baskets glow when an egg is selected, and the mouse hovers to show they're clickable
* Basket labels are now centered for any text
* Basket drakes are larger
* New challenges authored, to replace old style challenges and show new layout 

Fortunately, no changes were required to place 3 baskets instead of 4- they all get centered automatically.

kswenson commented on src/code/templates/fv-egg-sort-game.js:
```diff
@@ -76,8 +76,8 @@ let animationEvents = {
-          const leftOffset = 400,
-                topOffset = 200;
+          const leftOffset = 375,
+                topOffset = 100;
```
These (and other offsets like them) probably need to be unscaled. You can test it by checking with the offsets depend on the scale factor.

kswenson commented on src/code/templates/fv-egg-sort-game.js:
```diff
@@ -91,8 +91,8 @@ let animationEvents = {
-              leftOffset = -250,
-              topOffset = -40;
+              leftOffset = -255,
+              topOffset = -140;
```
These (and other offsets like them) probably need to be unscaled. You can test it by checking with the offsets depend on the scale factor.

ekosmin replied:
As far as I can tell, these should actually already be getting unscaled. Below, we pass in a scale factor, which goes to the `initialElementId` HOC and unscales the values. Something is definitely still off though, you saw the scaling doesn't work as expected

kswenson commented on src/code/templates/fv-egg-sort-game.js:
```diff
@@ -104,8 +104,8 @@ let animationEvents = {
-              leftOffset = 122,
-              topOffset = 204.5;
+              leftOffset = 70.5,
+              topOffset = 77.5;
```
These (and other offsets like them) probably need to be unscaled. You can test it by checking with the offsets depend on the scale factor.

kswenson commented on src/resources/authoring/gv2.json:
```diff
@@ -54,7 +54,7 @@
       "userChangeableGenes": "wings, forelimbs, hindlimbs"
     },
-    "eggSortingWings": {
+    "basicEggSortingWings": {
```
Have these naming conventions been run by Trudi? I believe there's a PT story to that effect.

ekosmin replied:
They haven't...let me ping her and ask her to fill them out in the Geniverse development master

kswenson commented on src/stylus/fablevision/fv-egg-sort-game.styl:
```diff
@@ -46,21 +46,28 @@
             right: 250px
             .basket-image
-              background: url('../resources/fablevision/layout-d/basket.png')
+              background: url('../resources/fablevision/layout-d/basket_inactive.png')
```
should be `background-image`

kswenson commented on src/stylus/fablevision/fv-egg-sort-game.styl:
```diff
@@ -46,21 +46,28 @@
               background-size: contain
               position: absolute
+            &.selected .basket-image
+              background: url('../resources/fablevision/layout-d/basket.png')
```
should be `background-image`

ekosmin commented:
I made the changes you suggested in my most recent commit. I also fixed the scaling issue that was happening- turns out I was unscaling the offsets when I actually didn't want to. Once scaling was fixed, it was relatively simple to fix animations from the eggs to the baskets, so I updated the offsets and egg sizes to get that working too.
